examples/org.eclipse.swt.snippets/src/org/eclipse/swt/snippets/SnippetExplorer.java

```diff
@@ -1042,7 +1042,7 @@ public void stopSnippets() {
 				if (!launchedSnippets.isEmpty()) {
 					System.err.println("Some Snippets are still running:");
 					for (Thread t : launchedSnippets) {
-						System.err.println("    " + t.getName() + " (ThreadId: " + t.getId() + ")");
+						System.err.println("    " + t.getName() + " (ThreadId: " + t.threadId() + ")");
```

Comment:
This is the warning fix that can lead to error if JRE is incorrectly set to Java 17 as outlined [here](https://github.com/eclipse-platform/eclipse.platform.swt/discussions/2840#:~:text=would%20see%20it.-,The%20method%20threadId()%20is%20undefined%20for%20the%20type%20Thread,-Once%20I%20finish)